File: algorithms/neetcode/algorithms/algorithms/src/main/java/com/mycompany/leetcode/easy/FindNumbersWithEvenNumberOfDigits/FindNumbersWithEvenNumberOfDigitsChappy1.java
package com.mycompany.leetcode.easy.FindNumbersWithEvenNumberOfDigits;

public class FindNumbersWithEvenNumberOfDigitsChappy1 {

    static int findNumbers(int[] nums) {
        int output = 0;
        for(int number : nums) {
            if(number == 100000) output++;
            else if (number > 10000) output += 0;
            else if(number >= 1000) output++;
            else if(number >= 100) output += 0;
            else if(number >= 10) output++;
        }
        return output;
    }

    public static void main(String[] args) {
        int[] nums = {12,345,2,6,7896};
        System.out.println(findNumbers(nums));
    }
}

File: algorithms/neetcode/algorithms/algorithms/src/main/java/com/mycompany/leetcode/easy/FindNumbersWithEvenNumberOfDigits/find-numbers-with-even-number-of-digits-chappy1.py

from typing import List

class S:
    def findNumbers(self, nums: List[int]) -> int:
        even_num_of_digits = 0
        
        for num in nums:
            if len(str(num)) % 2 == 0:
                even_num_of_digits += 1
        return even_num_of_digits

nums = [12,345,2,6,7896]
print(S().findNumbers(nums))